src/Providers/DataGrid/Wijmo/Helper/CellTemplateFactory.ts

```diff
@@ -9,12 +9,17 @@ namespace Providers.DataGrid.Wijmo.Helper.CellTemplateFactory {
     export function MakeCellTemplate(
         type: OSFramework.DataGrid.Enum.CellTemplateElementType,
         binding: string,
+        externalURL: string,
```

Comment:
As mentioned before, I think this should be the last optional parameter here.

Reply:
Done